File: system_b/educationalSystem_B/vo/courseVO.py

class Course:
    def __init__(self,id,name,credit,teacher,campus,share):
        self.id=id
        self.name=name
        self.credit=credit
        self.teacher=teacher
        self.campus = campus
        self.share = share

    def getId(self):
        return self.id

    def setId(self,id):
        self.id=id

    def getName(self):
        return self.name

    def setName(self,name):
        self.name=name

    def getCredit(self):
        return self.credit

    def setCredit(self,credit):
        self.credit=credit

    def getTeacher(self):
        return self.teacher

    def setTeacher(self,teacher):
        self.teacher=teacher

    def getCampus(self):
        return self.campus

    def setCampus(self,campus):
        self.campus=campus

    def getShare(self):
        return self.share

    def setShare(self,share):
        self.share=share

File: a_system/src/main/java/com/example/a_system/vo/CourseVO.java
package com.example.a_system.vo;

public class CourseVO {
    private String cno;
    private String cname;
    private String teacher;
    private String ctime;
    private String credit;
    private String share;
    private String ctype;

    public CourseVO(String cno, String cname, String teacher, String ctime, String credit, String share, String ctype) {
        this.cno = cno;
        this.cname = cname;
        this.teacher = teacher;
        this.ctime = ctime;
        this.credit = credit;
        this.share = share;
        this.ctype = ctype;
    }

    public String getCno() {
        return cno;
    }

    public void setCno(String cno) {
        this.cno = cno;
    }

    public String getCname() {
        return cname;
    }

    public void setCname(String cname) {
        this.cname = cname;
    }

    public String getTeacher() {
        return teacher;
    }

    public void setTeacher(String teacher) {
        this.teacher = teacher;
    }

    public String getCtime() {
        return ctime;
    }

    public void setCtime(String ctime) {
        this.ctime = ctime;
    }

    public String getCredit() {
        return credit;
    }

    public void setCredit(String credit) {
        this.credit = credit;
    }

    public String getShare() {
        return share;
    }

    public void setShare(String share) {
        this.share = share;
    }

    public String getCtype() {
        return ctype;
    }

    public void setCtype(String ctype) {
        this.ctype = ctype;
    }
}
